e2e/smoke/WORKSPACE.bazel

```diff
@@ -32,3 +36,22 @@ protoc_toolchains(
     name = "protoc_toolchains",
     version = "v25.3",
 )
+
```

Comment:
Starlarkification without Bzlmod is hopeless. How about disavowing compatibility with WORKSPACE + Bazel 8 and just bumping the rules_proto dep in `MODULE.bazel`, not here?

Reply:
Sure thanks @fmeum , giving that a try